keylime/signing.py

```diff
@@ -7,11 +8,17 @@
 from cryptography.hazmat.primitives.serialization import load_pem_public_key
 
 from keylime import keylime_logging
+from keylime.types import PathLike_str
 
 logger = keylime_logging.init_logging("signing")
 
 
-def verify_signature_from_file(key_file: str, filename: str, sig_file: str, description: str) -> None:
+def verify_signature_from_file(
+    key_file: Union[str, PathLike_str],
```

Comment:
ideally should all be `PathLike_str`, as always the meaning will be for key_file to indicate a file

on the other side the API of this function is not very good, as it is used for the side effect of raising an exception

Reply:
What do you mean? All callers should pass in a PathLike_str type?